src/styles/themes/selectBox.ts

```diff
@@ -0,0 +1,71 @@
+import type { ThemeOption } from '@types'
+
+type SelectBoxKeys = keyof typeof selectBox
+type SelectBoxValues = typeof selectBox[SelectBoxKeys]
+
+export type SelectBox = ThemeOption<SelectBoxKeys, SelectBoxValues>
+
+export const selectBox = {
+  medium: {
+    default: `
+      div:nth-child(1){ //triggerWrapper
```

Comment:
```suggestion
      div:nth-of-type(1){ //triggerWrapper
```
ssr에서는 style 구성요소도 함께 렌더링해서 nth-child라고 했을 때, 잠재적인 에러 원인이 될 수 있다고 합니다!
타입으로 명확하게 잡는 게 좋을 것 같아요!